region/lowernorfair/east/Mickey Mouse Room.json

```diff
@@ -989,6 +989,28 @@
             {"heatFrames": 60}
           ]}
         ]}
+      ],
+      "flashSuitChecked": true
+    },
+    {
+      "link": [4, 2],
+      "name": "Jump Into Respawning Block - Blocks Mostly Intact",
+      "requires": [
+        {"obstaclesCleared": ["B"]},
+        {"heatFrames": 600},
+        "canJumpIntoRespawningBlock",
+        {"or": [
+          "canInsaneJump",
```

Comment:
This one might be fine as is, though I wonder if it should maybe be bumped up a tier? (e.g. with `h_complexToCarryFlashSuit` for the lenient version, `h_trickyToCarryFlashSuit` for the one without lenience).

I found that it could be pretty reliable by shooting up immediately followed by 5 full-height jumps; some timing cue like this would probably be good to mention in the note.